docker-compose.yml

```diff
@@ -212,7 +213,7 @@ services:
   #############
 
   sql:
-    image: "mcr.microsoft.com/mssql/server"
+    image: "mcr.microsoft.com/mssql/server:2017-latest"
```

Comment:
I think this may be added by another change that has yet to be merged into main but Ill keep it in as it wont break anything.

Reply:
Yes it's in that envs PR - merged late yesterday.